index.html

```diff
@@ -200,53 +200,22 @@ <h1>About me</h1>
                         </ul>
                     </li>
                 </ul>
-                <!--<div class="tech">
-                    <div class="openable">
-                        <p>Languages</p>
-                        <img src="images/Techs/icons8-arrow-down-50.png">
-                    </div>
-                    <div class="techflex">
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-javascript-48.png">
-                            <p>JavaScript</p>
-                        </div>
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-html5-48.png">
-                            <p>HTML</p>
-                        </div>
-                        <div class="stackNotice">
-                            <img src="images/Techs/icons8-css3-48.png">
-                            <p>CSS</p>
-                        </div>
-                    </div>
-                </div>
-                <hr class="TechSeparation">
-                <div class="tech">
-                    <div class="openable">
-                        <p>Frameworks</p>
-                        <img src="images/Techs/icons8-arrow-right-50.png">
-                    </div>
-                </div>
-                <hr class="TechSeparation">
-                <div class="tech">
-                    <div class="openable">
-                        <p>Skills</p>
-                        <img src="images/Techs/icons8-arrow-right-50.png">
-                    </div>
-                </div>
-                <hr class="TechSeparation">-->
+
             </div>
         </section>
         <section class="comments">
             <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form class="form_comment">
-                <input type="text" id="fname" name="fname" class="input_comment" placeholder="Full Name"><br>
+            <form class="form_comment" action="https://formspree.io/f/xdovrwnp" method="post">
+                <input type="text" id="fname" name="fname" placeholder="Full Name" min="4" max="30" class="input_place input_comment" required>
+                <hr class="InputSeparation">
+                <br>
                 <br>
                 <br>
-                <input type="text" id="faddress" name="faddress" class="input_comment" placeholder="Email Address">
+                <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" type="email" min="10" max="30" required>
+                <hr class="InputSeparation">
```

Comment:
Great job using the required property 👍. However, you seem to have missed the maxlenght property for the name and the message as required. Kindly make sure you set the maxlenght of the name to 30 (maxlength="30") and the maxlenght of the message to 500 (maxlength="500"). This is not only part of the requirements but it will help your project look more professional 💯.

![require](https://github.com/alexansaa/Personal_Portafolio/assets/111214189/cd1c4e02-128c-4ab8-a122-ffc55a3a6e59)